### ISSUE TITLE
Search Widget crashes the application on iOS 9.2 and Safari

### ISSUE THREAD

**Issue by vcoelho** [Esri/Viewer#61]
Hello,

We did setup the following site(http://205.237.60.232:81/basicViewer/) using the latest version and we noticed that the search widget crashes when using the Safari Browser.

To reproduce the issue just open the site on Safari and iOS 9.2 and type "L"

**Comment by jgravois** [comment]
i'm not able to reproduce the crash you describe when visiting your own site.
Safari `9.0.2`
OSX `10.10.5`

**Comment by vcoelho** [comment]
Hello John,

The issue is only reproducible on iOS 9.2 and Safari.

**Comment by kellyhutchins** [comment]
@vcoelho  I was able to reproduce using your test app with ios 9.2 and Safari.  However I cannot reproduce it with a non customized version of the app: 
http://www.arcgis.com/apps/Viewer/index.html?webmap=f5b13dbed07c46cdb783cf361833aa6b

Did you make any code changes to the app?

**Comment by vcoelho** [comment]
Hello Kelly,

We only modified the default.js to add the WebMapId and the proxy url.

**Comment by bsvensson** [comment]
@vcoelho - you're using the 3.14 version, could you try with the latest (3.15)?

**Comment by vcoelho** [comment]
Hi Bjorn,

I'm experiencing the same results with the 3.15 version.

**Comment by mlrgisonline** [comment]
I'm experiencing the same issue in Safari. When I use the search window and tap on a result the application crashes. Here's the link to our web app: 

http://mtlandreliance.maps.arcgis.com/apps/Viewer/index.html?appid=4e76fba1a28a4db2aba9837ed4a031cf

Thanks,

**Comment by kellyhutchins** [comment]
This looks like a bug in the JavaScript API when zooming in to search results for Safari on ios. We'll work on getting this fixed in a future release - if you'd like to track this bug please submit an issue to Esri support for this so they can submit an official bug for the problem.

**Comment by vcoelho** [comment]
Hello, I did submit the case to Esri inc. Support once I have the BUG #  I will post it here.

Regards,

**Comment by kellyhutchins** [comment]
@vcoelho  great thank you!

**Comment by vcoelho** [comment]
The Bug was created by Tech Support : BUG-000093308:Search Widget crashes the application on iOS 9.2 and Safari.

**Comment by Biboba** [comment]
Same issue on IOS 8.2.
Cheers

**Comment by ezequias** [comment]
Have anyone a live demo with this widget?

**Comment by ezequias** [comment]
@vcoelho I can't see your application.

**Comment by ezequias** [comment]
@mlrgisonline what did you put to make the search auto complete? I thought this widget where only for geocoding.

**Comment by kellyhutchins** [comment]
Closing this issue since its due to a bug in the ArcGIS API for JavaScript. Once the issue is resolved in the api it will work in the template.

**Comment by Biboba** [comment]
FYI, I've just tested the new JS API 3.16 stating that :
BUG-000093308: Fixed an issue where the Search widget crashed applications using iOS 9.2 and Safari

But I am still having the issue. Does anyone have the same problem ?

Cheers

**Comment by cuzzup** [comment]
I am experiencing the same thing as well in ESRI's own Web AppBuilder for ArcGIS.  The Locate widget is causing Safari to crash.  If I use the app on an iPhone it works just fine, but if I attempt to use it on iPad running iOS 10.2.1 with Safari it crashes.  To verify it was in fact the Locate widget that was causing the crash I removed it from the web app and republished.  The web app no longer crashed in Safari when Locate widget was removed.  

Just the mere presence of the Locate widget in the app causes the crash in Safari, even if it is not actually engaged by the user.
